Uplift/AppDelegate.swift

```diff
@@ -27,7 +27,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         let defaults = UserDefaults.standard
         window?.rootViewController = defaults.bool(forKey: Identifiers.hasSeenOnboarding)
-            ? TabBarController()
+            ? TestViewController()//TabBarController()
```

Comment:
Have you already completed testing and, if so, shouldn't this be reverted to normal?